ui/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@ant-design/cssinjs": "^1.17.2",
     "antd": "^5.10.1",
+    "lodash": "^4.17.21",
```

Comment:
can we just install only the lodash that we need?

Reply:
changed to ```lodash.isequal```